diff --git a/node_modules/@payloadcms/db-mongodb/dist/index.js b/node_modules/@payloadcms/db-mongodb/dist/index.js
index 7d54942..55db8b7 100644
--- a/node_modules/@payloadcms/db-mongodb/dist/index.js
+++ b/node_modules/@payloadcms/db-mongodb/dist/index.js
@@ -69,7 +69,7 @@ function mongooseAdapter({ autoPluralization = true, connectOptions, disableInde
             url,
             versions: {},
             // DatabaseAdapter
-            beginTransaction: beginTransactionFunction,
+            beginTransaction: undefined,
             commitTransaction: _commitTransaction.commitTransaction,
             connect: _connect.connect,
             create: _create.create,
